### ISSUE TITLE
Does not show media, gives error when connecting to server

### ISSUE THREAD

**Issue by BENNIEV-ZA** [EraYaN/jellyfin-webng#44]
Myne will not connect to the server. Its a localhost. When I input my username and password it gives the following error
DEMO-STRING (your resumable items):

{"Items":[],"TotalRecordCount":0}

Did a electron windows 64 built.
on console it shows connection refused
I see its also connecting to emby addresses and not jellyfin.
I dont know how far the project is. If this is still down the road?

**Comment by anthonylavado** [comment]
Yes, this is still well down the road. Right now the only “published” code is just an authentication test that hasn’t been updated in a while.

Pull Requests are coming in now to bring it up to speed and start working on it.
